### ISSUE TITLE
moustache templates support

### ISSUE THREAD

**Issue by BloodyFoxy** [koalaman/shellcheck#1657]
I use [mustache](https://github.com/tests-always-included/mo) templates in my shell scripts. ShellCheck's behavior is very strange: I cannot even fully disable all warnings.

#### Here's what ShellCheck currently says:
Trying this: 
```bash
#!/bin/bash
{{MUSTACHE}}

```
Error output:
```
Line 2:
{{MOUSTACHE}}
^-- SC1009: The mentioned syntax error was in this brace group.
 ^-- SC1054: You need a space after the '{'.
 ^-- SC1073: Couldn't parse this brace group. Fix to allow more checks.
  ^-- SC1054: You need a space after the '{'.
           ^-- SC1083: This } is literal. Check expression (missing ;/\n?) or quote it.
            ^-- SC1083: This } is literal. Check expression (missing ;/\n?) or quote it.
 
Line 3:

^-- SC1056: Expected a '}'. If you have one, try a ; or \n in front of it.
^-- SC1072: Missing '}'. Fix any mentioned problems and try again.
```

Trying to disable mentioned SCs:

```bash
#!/bin/bash
# shellcheck disable=SC1009,SC1052,SC1054,SC1056,SC1072,SC1073,SC1083
{{MUSTACHE}}

```
Error output:
```
Line 3:
{{MOUSTACHE}}
^-- SC1009: The mentioned syntax error was in this brace group.
 ^-- SC1073: Couldn't parse this brace group. Fix to allow more checks.
 
Line 4:

^-- SC1072: Missing '}'. Fix any mentioned problems and try again.
```

#### Here's what I wanted or expected to see:
- [ ] Mustache templates support;
- [ ] No disabled SC in ShellCheck output.

**Comment by e-kwsm** [comment]
`{{MOUSTACHE}}` is not valid shell syntax at all, thus shellcheck cannot parse it.
